### PR TITLE
Fixes #13109 - seed bookmarks scoped per controller

### PR DESCRIPTION
**Pull request by ares**

### CODE REVIEW

File: db/seeds.d/15-bookmarks.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@
     { :name => "disabled", :query => 'status.enabled = false', :controller=> "hosts" },
     { :name => "ok hosts", :query => 'last_report > "35 minutes ago" and status.enabled = true and status.applied = 0 and status.failed = 0 and status.pending = 0', :controller=> "hosts" }
   ].each do |input|
-    next if Bookmark.find_by_name(input[:name])
+    next if Bookmark.where(:controller => input[:controller]).find_by_name(input[:name])
```
Comment (domcleal) [inline db/seeds.d/15-bookmarks.rb]: Nitpick, use where for both fields to avoid mixing the two where/find styles.

**Comment by ares** [comment]
changed to `where(a, b).first`, please rereview

**Comment by domcleal** [comment]
1) There's an existing error from #4151 where there's an incorrect controller listed for the seeded `eventful` bookmark that gets added in addition to the one renamed through db:migrate, but with the wrong controller (as it doesn't match).  Could you fix the controller name there please?

2) I think the audit_modified line is still having an effect on this, as it also isn't scoped to controller.  Deleting or modifying a bookmark on another controller with the same name will prevent it working.  Should this be scoped too?

**Comment by ares** [comment]
thanks, I think I've fixed both, please take another look

**Comment by domcleal** [comment]
Merged as b5a2de2, thanks @ares.

**Pull request closed without merging**